src/mkdocstrings/handlers/python.py

```diff
@@ -204,13 +204,13 @@ def collect(self, identifier: str, config: dict) -> dict:
             raise CollectionError(result["error"])
 
         if result["loading_errors"]:
-            for error in result["loading_errors"]:
-                log.warning(f"mkdocstrings.handlers.python: {error}")
+            for err in result["loading_errors"]:
+                log.warning(f"mkdocstrings.handlers.python: {err}")
 
         if result["parsing_errors"]:
-            for path, errors in result["parsing_errors"].items():
-                for error in errors:
-                    log.warning(f"mkdocstrings.handlers.python: {error}")
+            for path, problems in result["parsing_errors"].items():
+                for err in problems:
+                    log.warning(f"mkdocstrings.handlers.python: {err}")
```

Comment:
Same remark as before: I'd rather have a `# type: ignore` comment instead of using more variables.